SFML/SFML
# refactor: add missing `#include`s

----

## Description

When building the CMake target `SFML::Graphics`
with a sufficiently recent Clang using [Clang Modules](https://clang.llvm.org/docs/Modules.html)
(with flags `-stdlib=libc++ -fmodules -fbuiltin-module-map -fimplicit-module-maps`)
compilation fails with these (source, error) pairs:
```output
<prefix>/include/SFML/System/Time.inl:47:80: error: no member named 'milli' in namespace 'std'
   47 |     return std::chrono::duration_cast<std::chrono::duration<std::int32_t, std::milli>>(m_microseconds).count();
      |                                                                           ~~~~~^
<prefix>/src/SFML/Window/GlResource.cpp:43:52: error: declaration of 'move' must be imported from module 'std.utility.__utility.move' before it is required
   43 |     priv::GlContext::registerUnsharedGlObject(std::move(object));
      |                                                    ^
```

## Tasks


## How to test this PR?

Adding missing `#include`s shouldn't require changing how testing is done.

To test the current state of `master` as reported,
follow the description to setup Clang
and see the errors when building the CMake target `SFML::Graphics`.
